bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/Scene.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hue.internal;
+
+import java.lang.reflect.Type;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.types.StateOption;
+
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.reflect.TypeToken;
+
+/**
+ * Basic scene information.
+ */
+public class Scene {
+    public static final Type GSON_TYPE = new TypeToken<Map<String, Scene>>() {
+    }.getType();
+
+    private String id;
+    private String name;
+    @SerializedName("lights")
+    private List<String> lightIds;
```

Comment:
Minor remark: in the FullGroup class, we kept the field `lights`. We should just be coherent, I think.

Reply:
I personally would expect `getLights` to give me a list of `Light` not `String`. Not super important for me though, I also think that consistency is more important. Do you prefer it to stay with `lights`? Or should I change it to `lightIds` in `FullGroup`?

Reply:
I agree with you, it should rather be changed in the group class. I think I am the one who did this change (changing what was returned without renaming the method.